lib/cli/src/frameworks/react/ts/Welcome.tsx

```diff
@@ -0,0 +1,159 @@
+import React, {
+  AnchorHTMLAttributes,
+  ButtonHTMLAttributes,
+  DetailedHTMLProps,
+  FunctionComponent,
+  HTMLAttributes,
+} from 'react';
+
+type MainProps = Omit<DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>, 'style'>;
```

Comment:
Can we simplify the TypeScript in our welcome component? Otherwise it's very welcoming!

Reply:
I have in mind of having Welcome being a mdx story and having another more simple/friendly component.  Not sure how to simplify this one

Reply:
Agree with @shilman a simpler type would be great but I'm a React newbie so I don't if we can do better or if there is a workaround.